src/AccessibilityInsights.SharedUx/FileBug/BrowserEmulation.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using Microsoft.Win32;
+using System;
+using System.Diagnostics;
+
+namespace AccessibilityInsights.SharedUx.FileBug
+{
+    /// <summary>
+    /// This class is adapted from https://stackoverflow.com/questions/18333459/c-sharp-webbrowser-ajax-call
+    /// and sets several registry values in order to ensure that the hosted WebBrowser control renders
+    /// documents in standards mode. Without these changes the hosted WebBrowser is unable to properly
+    /// render some pages when filing issues
+    /// https://docs.microsoft.com/en-us/previous-versions/windows/internet-explorer/ie-developer/general-info/ee330730%28v%3dvs.85%29#browser-emulation
+    /// </summary>
+    public static class BrowserEmulation
+    {
+        public static void SetBrowserFeatureControl()
+        {
+            // http://msdn.microsoft.com/en-us/library/ee330720(v=vs.85).aspx
+
+            // FeatureControl settings are per-process
+            var fileName = System.IO.Path.GetFileName(Process.GetCurrentProcess().MainModule.FileName);
+
+            try
+            {
+                SetBrowserFeatureControlKey("FEATURE_BROWSER_EMULATION", fileName, GetBrowserEmulationMode());
+                SetBrowserFeatureControlKey("FEATURE_AJAX_CONNECTIONEVENTS", fileName, 1);
+                SetBrowserFeatureControlKey("FEATURE_GPU_RENDERING", fileName, 1);
+            }
+            catch (Exception)
+            {
+                // silently ignore
```

Comment:
>// silently ignore [](start = 16, length = 18)

Perhaps call ReportException on this Exception, so that we have some visibility that it's happening?